lib/format.js

```diff
@@ -1,69 +0,0 @@
-const path = require('path')
```

Comment:
+1 for removal.  I use an atom plugin to prettify as needed.

Reply:
To be clear, this doesn't remove it altogether, just simplifies it into just path through call to `prettier-eslint`